eng/scripts/Language-Settings.ps1

```diff
@@ -9,6 +9,7 @@ $GithubUri = "https://github.com/Azure/azure-sdk-for-java"
 $PackageRepositoryUri = "https://repo1.maven.org/maven2"
 
```

Comment:
When do you plan to use the new script you are adding to common in https://github.com/Azure/azure-sdk-tools/pull/6632?